php/utils.php

```diff
@@ -980,3 +980,26 @@ function is_bundled_command( $command ) {
 		? array_key_exists( $command, $classes )
 		: false;
 }
+
+/**
+ * Maybe prefix command string with "/usr/bin/env".
+ * Removes (if there) if Windows, adds (if not there) if not.
+ *
+ * @param string $command
+ *
+ * @return string
+ */
+function maybe_prefix_env( $command ) {
```

Comment:
I'm not a huge fan of functions starting with `maybe_*`, because it means their behavior is unpredictable.

Could we instead rename this to something like `force_env_on_nix_systems`?